transcoder/src/keyframes.go

```diff
@@ -70,25 +69,13 @@ func (kf *Keyframe) AddListener(callback func(keyframes []float64)) {
 	kf.info.listeners = append(kf.info.listeners, callback)
 }
 
-func (kf *Keyframe) Scan(src interface{}) error {
-	var arr pq.Float64Array
-	err := arr.Scan(src)
-	if err != nil {
-		return err
-	}
-	kf.Keyframes = arr
-	kf.IsDone = true
-	kf.info = &KeyframeInfo{}
-	return nil
-}
-
```

Comment:
This shouldn't be deleted! When we load from this from db (from `transcoder/src/metadata.go:191`), this won't set `IsDone` & won't know that only the array should be loaded from db.

Reply:
Yea I wasn't sure about this since it isn't explicitly referenced anywhere. Do you know how t his should be implemented with pgx? What type(s) will be provided as src?

Reply:
As-is as in with the existing `Scan` func, or without it?

If you mean that the scan function should remain as-is, what type should I replace `pq.Float64Array` with (so that the dep can be dropped)?